api/src/Controller/System/BaseSystem.php

```diff
@@ -26,5 +26,18 @@ public function __construct(Container $container)
     }
 
 
+    public static function install($container): void
+    {
+
+    }
+
+
+    public static function permissions($database): ?array
+    {
+        return ['api.get.log', 'api.get.configuration', 'api.get.log', 'api.put.configuration'];
```

Comment:
```suggestion
        return ['api.get.log', 'api.get.configuration', 'api.put.log', 'api.put.configuration'];
```

Looks like you might have meant `api.put.log` on the second one of these in the array.